src/api/java/io/github/cvc5/IOracle.java

```diff
@@ -0,0 +1,20 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Mudathir Mohamed
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2022 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * The cvc5 java API.
+ */
+
+package io.github.cvc5;
+
+public interface IOracle {
+  Term compute(Term[] terms) throws CVC5ApiException;
+}
```

Comment:
Format?

Reply:
This is how clang-format formats java interface.